media/static-bundles.json

```diff
@@ -1595,10 +1595,17 @@
     {
       "files": [
         "js/base/send-to-device.js",
+        "js/base/send-yourself.js",
         "js/firefox/whatsnew/whatsnew.js"
       ],
       "name": "firefox_whatsnew"
     },
+    {
+      "files": [
+        "js/base/send-yourself.js"
```

Comment:
I tried fixing the page locally, and I noticed the button on the thank-you message is unstyled:

![screenshot_2019-01-17 firefox lite 1](https://user-images.githubusercontent.com/400117/51308058-27850d80-1a39-11e9-81dc-89a1d2d509ec.png)

Reply:
I also had to include `"js/libs/spin.min.js"` in the bundle, which seems to be a dependency for `send-yourself` to work.